iomgr/test/sql_parser_test/SqlParserTest_Other.cpp

```diff
@@ -61,3 +75,31 @@ TEST(Query, ShowTables)
     // Check request type
     ASSERT_EQ(dbeRequest->m_requestType, requests::DBEngineRequestType::kShowTables);
 }
+
+TEST(Query, DescTable)
+{
+    // Parse statement and prepare request
+    const std::string statement("DESC TABLE SYS.SYS_TABLES");
+    parser_ns::SqlParser parser(statement);
+    parser.parse();
+
+    parser_ns::DBEngineSqlRequestFactory factory(parser);
+    const auto dbeRequest = factory.createSqlRequest();
+
+    // Check request type
+    ASSERT_EQ(dbeRequest->m_requestType, requests::DBEngineRequestType::kDescribeTable);
```

Comment:
This is good, but not enough. You must also check that there is correct database and table name.
For that, do something like this:
```cpp
    const auto& request = dynamic_cast<const requests::DescribeTableRequest&>(*dbeRequest);
    ASSERT_EQ(request.m_database, "MY_DATABASE");
```